common/assets/scss/components/_all.scss

```diff
@@ -1,22 +1,22 @@
-@import "autocomplete";
-@import "focus-table";
-@import "icons";
-@import "sortable-table";
-@import "stack-trace";
-@import "sticky-sidebar";
+@import 'autocomplete';
```

Comment:
Any reason for these quote changes? I feel like every other `scss` file in our app uses double quotes so this one would now stick out.

Reply:
As noted in the next comment...  (tldr; no, not really)